data_engineering/derive_pipeline_runner.py

```diff
@@ -1,8 +1,6 @@
-import pandas as pd
-
 from utils import pipeline_io
 
-from data_engineering.derive.derive import get_raw_values
+from data_engineering.utils.derive import get_raw_values
```

Comment:
utils itself is a library already, so no need to do `from data_engineering.utils` you can do `from utils` directly. 

In pycharm you might need to set dir `data_engineering` as source path -> right click, mark dir as;

Reply:
yea this makes sense if we only run it from pycharm? if we run it from terminal then we have to manually set PYTHONPATH to include `data_engineering` as source path?

Reply:
no you don't need to - that's the beauty of making this a library XD
mark as source dir is only for pycharm for some reason..